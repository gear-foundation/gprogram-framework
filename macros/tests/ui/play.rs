use gprogram_framework_macros::command_handlers;

command_handlers!();

fn main() {}
