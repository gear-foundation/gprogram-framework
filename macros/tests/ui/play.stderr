error: No command handlers found
 --> tests/ui/play.rs:3:1
  |
3 | command_handlers!();
  | ^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `command_handlers` (in Nightly builds, run with -Z macro-backtrace for more info)
